pytransform3d/test/test_transform_manager.py

```diff
@@ -300,9 +300,12 @@ def test_from_to_dict():
 
     tm_dict = tm.to_dict()
     tm2 = TransformManager.from_dict(tm_dict)
```

Comment:
requesting a transformation from tm2 should also raise an exception before this patch, can you confirm this?

Reply:
Yes that is the case. You can try by checking out 28ecbefc0b9c5e6ec48f65533206ab9919f88898 and running the tests.